nvm.sh

```diff
@@ -1117,10 +1117,10 @@ nvm_compute_checksum() {
     command gsha256sum "${FILE}" | command awk '{print $1}'
   elif nvm_has "openssl" && ! nvm_is_alias "openssl"; then
     nvm_err 'Computing checksum with openssl dgst -sha256'
-    command openssl dgst -sha256 "${FILE}" | rev | command awk '{print $1}' | rev
+    command openssl dgst -sha256 "${FILE}" | command rev | command awk '{print $1}' | command rev
```

Comment:
i'm not sure about the `rev` ones - the checksum commands are relatively new, so i'm not sure if we'd likely get a bug report, but probably best to revert them for now.